README.md

```diff
@@ -269,9 +269,11 @@ MESH configuration is done using environment variables:
 | NHAIS_MESH_MAILBOX_PASSWORD      | N/A                       | The password for MAILBOX_ID
 | NHAIS_MESH_SHARED_KEY            | N/A                       | Shared key used to generate auth token. Provided by MESH operator (OpenTest, PTL, etc)
 | NHAIS_MESH_HOST                  | N/A                       | Hostname of MESH service
+| NHAIS_MESH_CERT_VALIDATION       | N/A                       | Certificate validation for SSL connection
```

Comment:
Default: false

> "true" enables certificate validation for SSL connection

Reply:
Changed out mind :)

Default: true

> "false" disable certificate validation for SSL connection

Reply:
done, src/main/resources/application.yml & src/recepResponder/resources/application.yml
default = true.
It should be set to false when running dockers vs fake-mesh locally